templates/components/buttons.html.twig

```diff
@@ -7,20 +7,12 @@
     {{ _self.action_cardtoolbutton('collapse', {collapse: target, title: label|default('')}) }}
 {% endmacro %}
 
-{# TODO test me #}
```

Comment:
wasn't used anywhere, so I removed it for now

Reply:
👍